### ISSUE TITLE
Build failure in `exe:brick-markup-demo` with `vty-5.34`

### ISSUE THREAD

**Issue by sjakobi** [jtdaugherty/brick#361]
```
programs/MarkupDemo.hs:28:49: error:
    • No instance for (Monoid V.Attr) arising from the literal ‘", "’
    • In the first argument of ‘(<>)’, namely ‘", "’
      In the second argument of ‘(<>)’, namely
        ‘", " <> ("world!" @@ fg V.red)’
      In the second argument of ‘($)’, namely
        ‘("Hello" @@ fg V.blue) <> ", " <> ("world!" @@ fg V.red)’
   |
28 |         m1 = markup $ ("Hello" @@ fg V.blue) <> ", " <> ("world!" @@ fg V.red)
   |                                                 ^^^^
cabal: Failed to build exe:brick-markup-demo from brick-0.67.
```

As a Hackage trustee, I have created a Hackage revision that should prevent users from encountering this issue: https://hackage.haskell.org/package/brick-0.67/revisions/

**Comment by jtdaugherty** [comment]
Whoops. Thank you for letting me know about this!

**Comment by jtdaugherty** [comment]
I've released 0.68 which addresses this. I might put in revisions for older versions, though, since this issue will come up for any older version of brick that tries to use Vty 5.34. Thanks again!
